Docs/reference/content/reference/driver/crud/sessions-and-transactions.md

```diff
@@ -0,0 +1,228 @@
+
+1. StartTransaction in the options parameter
+2. StartSession in the defaultTransactionOptions parameter
+3. Defaulted
+
+You can specify different options at different levels. An option specified in StartTransaction overrides the same option specified in the defaultTransactionOptions passed to StartSession, which in turn overrides the default value.
+
+#### TransactionOptions
+
+```csharp
+public class TransactionOptions
+{
+    public ReadConcern ReadConcern { get; };
+    public ReadPreference ReadPreference { get; };
+    public WriteConcern WriteConcern { get; };
+
+    public TransactionOptions(
+        Optional<ReadConcern> readConcern = default(Optional<ReadConcern>),
+        Optional<ReadPreference> readPreference = default(Optional<ReadPreference>),
+        Optional<WriteConcern> writeConcern = default(Optional<WriteConcern>));
+
+    public TransactionOptions With(
+        Optional<ReadConcern> readConcern = default(Optional<ReadConcern>),
+        Optional<ReadPreference> readPreference = default(Optional<ReadPreference>),
+        Optional<WriteConcern> writeConcern = default(Optional<WriteConcern>))
+}
+```
+
+Create an instance of TransactionOptions either by calling the constructor with any desired optional arguments, or by calling the With method on an existing instance of TransactionOptions to create a new instance with some values changed.
+
+##### ReadConcern
+
+The ReadConcern used while in the transaction. All operations in the transaction use the ReadConcern specified when StartTransaction is called.
+
+##### ReadPreference
+
+The ReadPreference used while in the transaction. Currently, the ReadPreference for a transaction must be Primary.
+
+##### WriteConcern
+
+The WriteConcern used for this transaction. The WriteConcern only applies when committing the transaction, not to the individual operations executed while in the transaction.
+
+### CommitTransaction and CommitTransactionAsync
+
+```csharp
+void CommitTransaction(CancellationToken cancellationToken = default(CancellationToken));
+Task CommitTransactionAsync(CancellationToken cancellationToken = default(CancellationToken));
+```
+
+You must call CommitTransaction or CommitTransactionAsync in order for a transaction to be committed. If a session is ended while a transaction is in progress the transaction will be automatically aborted.
+
+The commit is executed with the WriteConcern specified by the transaction options.
+
+### AbortTransaction and AbortTransactionAsync
+
+```csharp
+void AbortTransaction(CancellationToken cancellationToken = default(CancellationToken));
+Task AbortTransactionAsync(CancellationToken cancellationToken = default(CancellationToken));
+```
+
+Call AbortTransaction or AbortTransactionAsync to abort a transaction. Since any transaction in progress when a session is ended is automatically aborted you can also implicitly abort an uncommitted transaction by simply ending the session.
```

Comment:
I found this line to be mildly confusing. Perhaps:
"Since any transaction in progress is automatically aborted when a session is ended, you can also implicitly abort...."?

Reply:
Revised